python/mxnet/__init__.py

```diff
@@ -84,22 +86,20 @@
 from . import test_utils
 
 from . import rnn
-
 from . import gluon
 
-# Dynamic library module should be done after ndarray and symbol are initialized
-from . import library
-from . import tvmop
-
-__version__ = base.__version__
-
+# use mx.kv as short for kvstore
 # Dist kvstore module which launches a separate process when role is set to "server".
```

Comment:
Maybe you also want to update the comment here. In the case of HVD, there will be no "server" process.

Reply:
Sure. I think there will be a separate PR for unifying the launching method of different backends, but yes I can add some clarification here in the doc first